Source/WebCore/platform/graphics/coretext/FontCoreText.cpp

```diff
@@ -780,6 +780,22 @@ FloatRect Font::platformBoundsForGlyph(Glyph glyph) const
     return boundingBox;
 }
 
+Vector<FloatRect> Font::platformBoundsForGlyphs(const Vector<Glyph>& glyphs) const
+{
+    Vector<CGRect> rectsForGlyphs(glyphs.size());
+    auto ignoredBoundingBox = CTFontGetBoundingRectsForGlyphs(getCTFont(), platformData().orientation() == FontOrientation::Vertical ? kCTFontOrientationVertical : kCTFontOrientationHorizontal, glyphs.data(), rectsForGlyphs.data(), rectsForGlyphs.size());
```

Comment:
I believe we are not using `ignoredBoundingBox` so probably we don't need to store it.